client/context/context.go

```diff
@@ -34,6 +28,7 @@ type CLIContext struct {
 	Codec         *codec.Codec
 	AccDecoder    auth.AccountDecoder
 	Client        rpcclient.Client
+	TxBldr        authtxb.TxBuilder
```

Comment:
This is one of the major changes in this PR. The `CLIContext` now has a `TxBuilder`. Many of the functions defined in `client/utils` and `client/context` took both the `CLIContext` and `TxBuilder` as arguments. This PR changes those functions to be defined on `CLIContext` and use the `cliCtx.TxBldr`. Incidentally this reduces boilerplate for writing CLI functions, makes function signatures much easier to understand, and makes the flow for transaction generation (i.e. offline vs online) much more comprehensible.